target/linux/octeon/base-files/lib/upgrade/platform.sh

```diff
@@ -1,123 +1,125 @@
-#
-# Copyright (C) 2014 OpenWrt.org
-#
```

Comment:
As stated before, please do not remove any copyrights (ever)!

Reply:
sure